client_code/historic/logging.py

```diff
@@ -0,0 +1,16 @@
+# SPDX-License-Identifier: MIT
+# Copyright (c) 2021 anvilistas
+from anvil_extras import logging
+
+__version__ = "0.0.1"
+
+LOGGER = logging.Logger("anvil_labs.historic", level=logging.INFO)
+
+
+def set_debug(with_debug):
+    levels = {True: logging.DEBUG, False: logging.INFO}
+    LOGGER.level = levels[with_debug]
```

Comment:
Show me some python source code with this pattern and I'll buy into it 😉

Reply:
me too - once I actually write it properly and stop just letting copilot do its thing!